tests/db/test_bigchain_api.py

```diff
@@ -542,6 +542,31 @@ def test_valid_non_create_transaction_after_block_creation(self, b, user_vk, use
         assert tx_valid_signed == b.validate_transaction(tx_valid_signed)
         assert tx_valid_signed == b.is_valid_transaction(tx_valid_signed)
 
```

Comment:
@diminator or @r-marques Not sure who wrote these tests, but the big change here is every `tx` has to be put into a block and voted valid.